posts/inside-rust/2022-02-22-compiler-team-ambitions-2022.md

```diff
@@ -382,7 +377,7 @@ The compiler team leadership plans to put out a post in June summarizing the pro
 
 #### I did not see any mention of monadic burritos (or other non-Rust language feature); why is that not part of your plan?
 
-The scope of this doc is largely restricted to Compiler Team issues. Language design work is done by the Language Design team. You can reach out to them about their initiatives for this year and beyond.
+The scope of this doc is largely restricted to Compiler Team issues. Language design work is done by the Language Team. You can reach out to them about their initiatives for this year and beyond.
```

Comment:
technically T-lang does refer to itself as "the language design team", since one can argue that "language" encompasses design *and implementation* (i.e. T-compiler is the latter).

But I don't actually care, and I think the text is indeed improved with your change (it sounds less snarky with your wording, even if it is less quote-unquote precise).

Reply:
Ah, ok good to know, thanks! The repeated use of "language design" in such close proximity was bothering me and rust-lang.org/governance just refers to it as the "language team" so that's what I went with.

Reply:
Is the lang team planning to write some kind of blog post like this? If so, perhaps we can say something like
 
```suggestion
The scope of this doc is largely restricted to Compiler Team issues. The Language Team is planning to write more about their initiatives for this year and beyond in another post. Stay tuned for that!
```